src/renderer/services/slippi/slippi.service.ts

```diff
@@ -25,45 +29,54 @@ const handleErrors = (errors: readonly GraphQLError[] | undefined) => {
 };
 
 class SlippiBackendClient implements SlippiBackendService {
-  private httpLink: HttpLink;
   private client: ApolloClient<NormalizedCacheObject>;
 
   constructor(
     private readonly authService: AuthService,
     private readonly dolphinService: DolphinService,
     clientVersion?: string,
   ) {
-    this.httpLink = new HttpLink({ uri: SLIPPI_BACKEND_URL });
-    this.client = new ApolloClient({
-      link: this.httpLink,
-      cache: new InMemoryCache(),
-      name: "slippi-launcher",
-      version: clientVersion,
-    });
-  }
+    this.authService = authService;
```

Comment:
```suggestion
```
We don't need this. Giving an accessibility modifier on a constructor param already assigns it to be accessible from within the class. i.e. `private authService: AuthService` already implies `this.authService = authService`.